packages/backend/Cargo.toml

```diff
@@ -8,7 +8,7 @@ exclude = ["/bindings", "/api-wrapper"]
 [dependencies]
 tokio = { version = "1.12.0", default-features = false }
 once_cell = { version = "1.8.0", default-features = false }
-iota-wallet = { git = "https://github.com/iotaledger/wallet.rs", rev = "054c36fd3281119b22c09da25e175a3d08465873", default-features = false, features = ["stronghold", "ledger-nano", "ledger-nano-simulator", "participation"] }
+iota-wallet = { git = "https://github.com/iotaledger/wallet.rs", rev = "79163acbecbf2aeecb8caa99768f4d1437644bed", default-features = false, features = ["stronghold", "ledger-nano", "ledger-nano-simulator", "participation"] }
```

Comment:
We need to be careful when merging this and #1918 (it also changes the wallet.rs commit hash we use).